wcfsetup/install/files/lib/system/importer/UserImporter.class.php

```diff
@@ -238,19 +250,20 @@ public function import($oldID, array $data, array $additionalData = [])
             ]);
         }
 
-        // save default user events
-        $sql = "INSERT IGNORE INTO  wcf" . WCF_N . "_user_notification_event_to_user
-                                    (userID, eventID)
-                VALUES              (?, ?)";
-        $statement = WCF::getDB()->prepareStatement($sql);
-        WCF::getDB()->beginTransaction();
-        foreach ($this->eventIDs as $eventID) {
-            $statement->execute([
-                $user->userID,
-                $eventID,
-            ]);
+        if (!empty($this->eventIDs)) {
```

Comment:
I'm pretty sure that `$this->eventIDs` can never be empty, because even the Core itself provides some.

Reply:
They can, because they are filtered for `preset = 1` and the admin might have removed all presets.